components/eventing-controller/pkg/env/nats_config.go

```diff
@@ -25,8 +25,14 @@ type NatsConfig struct {
 	IdleConnTimeout     time.Duration `envconfig:"IDLE_CONN_TIMEOUT" default:"10s"`
 
 	// JetStream-specific configs
-	JSStreamName            string `envconfig:"JS_STREAM_NAME" default:"kyma"`
-	JSStreamStorageType     string `envconfig:"JS_STREAM_STORAGE_TYPE" default:"memory"`
+	// Name of the JetStream stream where all events are stored.
+	JSStreamName string `envconfig:"JS_STREAM_NAME" default:"kyma"`
+	// Storage type of the stream, memory or file.
+	JSStreamStorageType string `envconfig:"JS_STREAM_STORAGE_TYPE" default:"memory"`
+	// Retention policy specifies when should events be deleted from the stream.
+	//  interest: when all known observables have acknowledged a message it can be removed.
+	//  limits: messages are retained until any given limit is reached. This can be
```

Comment:
```suggestion
	//  limits: messages are retained until any given limit is reached.
```
I don't love splitting sentences across lines

Reply:
Thank you for the feedback. I think the code comments don't need review. Only the README file.

Reply:
It's not mandatory but I don't think code in text should stay unreviewed.